jenkins/opensearch-dashboards/integ-test.jenkinsfile

```diff
@@ -363,6 +363,14 @@ pipeline {
         always {
             node(AGENT_LABEL) {
                 script {
+                    def triggerIntegTestNotification =
+                            build job: 'integ-test-notification',
+                            propagate: false,
+                            wait: false,
+                            parameters: [
+                                string(name: 'INPUT_MANIFEST', value: "${env.version}/opensearch-dashboards-${env.version}.yml"),
+                                string(name: 'DISTRIBUTION_NUMBER', value: "${env.buildId}")
```

Comment:
`DISTRIBUTION_BUILD_ID`?

Reply:
Will take up as an enhancement. Will need to make changes in build-lib. :(